Backpack-Common/Carousel/Classes/BPKInternalCarousel.swift

```diff
@@ -24,36 +24,45 @@ public protocol BPKInternalCarouselDelegate: AnyObject {
 }
 
 public class BPKInternalCarousel: UIView {
+    public enum PageIndicatorVisibility {
+        case hidden
+        case visible(_ paddingBottom: CGFloat)
```

Comment:
should this be a BPKSpacing value?

Reply:
ah, no this can't be BPKSpacing, this is in Backpack-Common! BPKSpacing are defined in UIKit and SwiftUI, so from this class point of view, BPKSpacing don't exist!